src/gatsby-plugin-theme-ui/index.js

```diff
@@ -0,0 +1,10 @@
+export default {
```

Comment:
Is it used for slides?

Reply:
Yes slides need that but it will be applied to all pages